src/lib/targeting.ts

```diff
@@ -46,12 +44,13 @@ export const shouldThrottle = (
     }
 
     const viewsInThrottleWindow = views.filter(view => {
-        return daysSince(new Date(view.date), now) <= config.maxViewsDays;
+        return daysSince(new Date(view.date), now) < config.maxViewsDays;
     });
 
-    const exceedsViewsInWindow = viewsInThrottleWindow.length >= config.maxViewsCount;
+    const exceedsViewsInWindow = viewsInThrottleWindow.length > config.maxViewsCount;
```

Comment:
Given `maxViewsCount=4`...
We want the throttle to apply when `viewsInThrottleWindow.length` equals `5`.